CHANGES.md

```diff
@@ -1,6 +1,13 @@
 Release Changes List
 ====================
 
+version 2.0.1
+=============
+
+This version is an update to the 2.0 release (just below) to address a
+bug on InfiniBand systems using LLVM 16 or 17.
+
+
 version 2.0
```

Comment:
Small nit, but I would say "portability bug", and maybe clarify that this only affects gasnet+ibv